Lib/fontTools/cffLib/cff2mergePen.py

```diff
@@ -0,0 +1,297 @@
+# Copyright (c) 2009 Type Supply LLC
```

Comment:
this header was probably copy pasted by mistake

Reply:
Fixed.